### PR TITLE
`General`: Remove participant score pages

### PR DESCRIPTION
**Pull request by ge65cer**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Server
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/server/).
- [x] I implemented the changes with a good performance and prevented too many database calls.
- [x] I documented the Java code using JavaDoc style.
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

The _Participant Score_ pages, for a course or an exam, do not offer any real uses cases or benefits for instructors (see discussion in #6230). They are mainly for debugging, as the student scores are shown on another page.
Example:
- https://artemis-test2.artemis.in.tum.de/course-management/174/participant-scores
- https://artemis-test2.artemis.in.tum.de/course-management/174/exams/65/participant-scores

### Description

Removed the _Participant Score_ pages for courses and exams.

Decluttered the Course Management header by grouping some less important buttons into a dropdown.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Check the 'Participant Scores' button is no longer there
4. Ensure everything else, especially the 'Scores' page, is still working as expected

#### Exam Mode Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Go to an existing exam with student submissions (or create one)
3. Ensure that the 'Scores' page still works as expected

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2
#### Exam Mode Test
- [x] Test

### Test Coverage

_Unchanged._

### CODE REVIEW

File: src/test/javascript/spec/component/exam/exam-scores/exam-scores.component.spec.ts [hunk 1]
```diff
@@ -899,24 +899,6 @@ describe('ExamScoresComponent', () => {
         });
     });
 
-    it('should not delegate user if authorisation is not sufficient', () => {
```
Comment (Strohgelaender) [inline src/test/javascript/spec/component/exam/exam-scores/exam-scores.component.spec.ts:902]: unused imports in this file.

**Review by JohannesStoehr — CHANGES_REQUESTED** [review]
Can you give the dropdown a suiting name that is displayed next to the arrow?
I would currently expect some more context regarding the assessment and not scores or plagiarism cases

**Comment by ge65cer** [comment]
@JohannesStoehr It's a split button where I put everything related to student submissions. Would you prefer a separated dropdown?

**Review by Strohgelaender — CHANGES_REQUESTED** [review]
Please also remove the related server endpoints.

**Comment by JohannesStoehr** [comment]
> @JohannesStoehr It's a split button where I put everything related to student submissions. Would you prefer a separated dropdown?

In that case I would expect a dropdown labeled "Student Scores/Assessment" and clicking on one of the options would then redirect to that page

**Comment by ge65cer** [comment]
@JohannesStoehr That would have the downside that the Assessment Dashboard would no longer accessible with one click.

**Comment by ge65cer** [comment]
Server tests fail due to coverage. I think it is safe to decrease the threshold in this case, @krusche?
(This PR only removes code.)

**Review by LiaLism — APPROVED** [review]
Everything worked as expected. Tested on ts0.

**Review by timo07k — DISMISSED** [review]
Everything works as expected. Tested on TS1.

**Review by luisa1999 — APPROVED** [review]

**Review by Vinicius-Agreste — APPROVED** [review]
Tested on ts2. Worked as expected. Looks good.

**Review by PhilipWerz — DISMISSED** [review]
Everything works as expected. Dropdown looks good. Tested on TS1.

Remark:
Score Button still visible in "Course Management" Overview Page. (According to the description not part of the PR thouh)

**Comment by krusche** [comment]
> @JohannesStoehr That would have the downside that the Assessment Dashboard would no longer be accessible with one click.

Can we add links to `scores`, `plagiarism cases` and `grading key` on the assessment dashboard page?

**Comment by ge65cer** [comment]
screenshot

**Review by JohannesStoehr — APPROVED** [review]
I like the new buttons in the assessment dashboard. Looks good to me now

**Pull request merged**